docs/pages/installation.md

```diff
@@ -1,59 +1,195 @@
 ---
 title: Installation
-description: There are many ways to install Foundation, but if you're just getting started, we have a few suggestions.
+description: There are many ways to install Foundation from downloading the compiled CSS, automated Sass installs to popular package managers.
 video: '6KwsGcEHVTE'
 ---
 
-### Command-Line Tool
+## How to choose the right version for you?
 
-Not a fan of GUIs? The Node-powered Foundation CLI can install the same template projects for you. Install it with npm:
+Each of these different ways to get started serves a different skill level or preferred workflow. If this is all new to you, don’t worry; The video above will give you an overview of each method and help you decide which is best for you.
 
-```bash
-npm install --global foundation-cli
-```
+Here’s a breakdown of the options you’ll see on this page:
```

Comment:
This breakdown is way better, but I think a little bit intimidating, especially if you're brand new to web development. Can we highlight which one we recommend for a complete beginner?